src/components/PageLayout.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function PageLayout({ children }) {
```

Comment:
do we need layout. it is simple project. Page with component should be enough. But you can choose.

Reply:
Yeah, it's little bit redundant but when having 15 pages in the project it will make code base look prettier.